docs/troubleshooting/index.mdx

```diff
@@ -5,6 +5,10 @@ slug: /troubleshooting
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
+:::tip 
```

Comment:
Perhaps we should add it to behavior? I think it's something users consider before installing in a real cluster...